src/test/java/org/opensearch/forecast/model/ForecastResultTests.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.forecast.model;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import org.junit.Before;
+import org.opensearch.commons.authuser.User;
+import org.opensearch.core.xcontent.ToXContent;
+import org.opensearch.test.OpenSearchTestCase;
+import org.opensearch.timeseries.TestHelpers;
+import org.opensearch.timeseries.model.Entity;
+import org.opensearch.timeseries.model.FeatureData;
+
+public class ForecastResultTests extends OpenSearchTestCase {
+    List<ForecastResult> result;
+
+    @Override
+    @Before
+    public void setUp() throws Exception {
+        super.setUp();
+        // Arrange
+        String forecasterId = "testId";
+        long intervalMillis = 1000;
+        Double dataQuality = 0.9;
+        List<FeatureData> featureData = new ArrayList<>();
+        featureData.add(new FeatureData("f1", "f1", 1.0d));
+        featureData.add(new FeatureData("f2", "f2", 2.0d));
+        long currentTimeMillis = System.currentTimeMillis();
+        Instant instantFromMillis = Instant.ofEpochMilli(currentTimeMillis);
```

Comment:
Why not just use `Instant.now()` here instead of truncating to milliseconds?  (I know, it doesn't matter for a 10 second interval, but it's less text!)

Reply:
It is because we are saving time in the unit of milliseconds. Please take a look at Forecast.toXContent method:

``
if (executionEndTime != null) {
            // can be null during preview
            xContentBuilder.field(CommonName.EXECUTION_END_TIME_FIELD, executionEndTime.toEpochMilli());
        }
```